### PR TITLE
Add pagination / filtering to active users table in Active Sessions

### LINKED ISSUE #54: Use pagination for listing active user sessions
This task has actually been taken from [an Alfresco JIRA ticket](https://issues.alfresco.com/jira/browse/REPO-1077) regarding inclusion of the original Support Tools addon in Alfresco Enterprise. When many users are logged in the list will be too large to display properly and might take a long time to load. There should be pagination to only load subsets of the list (like we do for the dashboard components / user dashboards on the Share-tier).

### PR DESCRIPTION
**Pull request by AFaust**
### CHECKLIST

- [x] There aren't existing pull requests attempting to address the issue mentioned here
- [x] Submission developed in a feature branch--not master

### CONVINCING DESCRIPTION

This PR refactors the table of active users listed in the Active Sessions tool to use (UI-only) pagination and filtering to improve usability for administrators and avoid scaling the page to an extreme hight, especially in larger systems.
As a minor aside, the JSON response for the active sessions graphs has been altered to not quote numerical values. We should standardise on a clean JSON style and I already used a similar style in #96. The web script backend for listing / logging out active users has been refactored to use URIs that are more resource oriented, and also appropriate HTTP verbs (e.g. "delete" for active user logout).

### RELATED INFORMATION

Fixes #54

### CODE REVIEW

File: repository/src/main/amp/config/alfresco/templates/webscripts/org/orderofthebee/support-tools/admin/ootbee-support-tools/active-sessions.get.html.ftl [hunk 1]
```diff
@@ -38,101 +38,104 @@ Copyright (C) 2005-2016 Alfresco Software Limited.
...
+        <div class="column-right">
+            <@field value=connectionPoolData.driverClassName label=msg("activesessions.database.driver")?html />
+            <@field value=connectionPoolData.url label=msg("activesessions.database.url")?html />
+            <@field value=connectionPoolData.initialSize?c label=msg("activesessions.database.initialSize")?html />
+            <@field value=connectionPoolData.maxActive?c label=msg("activesessions.database.maxActive")?html />
+            <@field value=connectionPoolData.minIdle?c label=msg("activesessions.database.minIdle")?html />
+            <@field value=connectionPoolData.maxIdle?c label=msg("activesessions.database.maxIdle")?html />
+          
+            <div class="control field">
+                <div style="background: #7fff7f; width:0.6em; height:0.7em; border:1px solid #00ff00; display:inline-block;"></div>
```
Comment (binduwavell) [inline repository/src/main/amp/config/alfresco/templates/webscripts/org/orderofthebee/support-tools/admin/ootbee-support-tools/active-sessions.get.html.ftl]: I know this is just a refactor, so the following comment is more general than this line. I suspect it makes sense to externalize any inline styles like this into our CSS, generally named classes are much easier to comprehend than these long strings of instructions :)

File: repository/src/main/amp/web/ootbee-support-tools/js/active-sessions.js [hunk 2]
```diff
@@ -1,48 +1,46 @@
...
  */
 /*
- * Linked to Alfresco Copyright
- * (C) 2005-2016 Alfresco Software Limited.
+ * Linked to Alfresco
+ * Copyright (C) 2005-2017 Alfresco Software Limited.
  */
 
-/* global Admin: false, el: false, TimeSeries: false, SmoothieChart: false */
+/* global Admin: false, el: false, TimeSeries: false, SmoothieChart: false, $: false */
```
Comment (binduwavell) [inline repository/src/main/amp/web/ootbee-support-tools/js/active-sessions.js]: What is this?

File: repository/src/main/amp/config/alfresco/templates/webscripts/org/orderofthebee/support-tools/admin/ootbee-support-tools/active-sessions.get.html.ftl (hunk 1 above)
Comment (AFaust) [reply repository/src/main/amp/config/alfresco/templates/webscripts/org/orderofthebee/support-tools/admin/ootbee-support-tools/active-sessions.get.html.ftl]: Fully agreed - externalised style beats inline style 90% of the time.

File: repository/src/main/amp/web/ootbee-support-tools/js/active-sessions.js (hunk 2 above)
Comment (AFaust) [reply repository/src/main/amp/web/ootbee-support-tools/js/active-sessions.js]: This is a directive for the JSHint validator included in our project POM, telling the validator which global variables we expect to be available (and thus not cause an "undefined variable" issue). The dollar sign is the shorthand for jQuery (I know, I don't like this handle as well, but it's what jQuery has been using for a decade now I believe).

**Pull request merged**